app/assets/stylesheets/components/left-nav-sub-content.scss

```diff
@@ -109,7 +109,7 @@
           color: $cu-red;
           width:100%;
           padding: 0;
-          font-size: 18px;
+          font-size: 22px;
```

Comment:
Minor question, but is it worth using rem for new changes?

Reply:
@nicholasnadel probably but this is just for a little triangle icon and there are a bunch of other pixel settings in this stylesheet that I wasn't planning on changing and testing right now. Just trying to get a quick fix in because the current style and sizes for the Dept Contact widget are wacky. Can we come back to that?